src/rpc/nft.ts

```diff
@@ -0,0 +1,58 @@
+import { plainToClass } from 'class-transformer';
+import { getWretcher } from './wretch';
+import { trustedNodeBaseUrl, baseUrl, rpcVersion } from './config';
+import { getUserProperty, getSpecificDayOfMonth } from '../utils';
+import { NFTDailyStat } from '../types';
+import configstore from '../common/configstore';
+
+export class NFTHttp {
+  private server: string;
+  private rpcVersion: string;
+  private trustedNodeBaseUrl: string;
+
+  public constructor(
+    server: string,
+    version: string,
+    trustedNodeBaseUrl: string,
+  ) {
+    this.server = server + '/nft/' + version + '/stats';
+    this.rpcVersion = version;
+    this.trustedNodeBaseUrl = trustedNodeBaseUrl;
+  }
+
+  public async dailyUsage(): Promise<NFTDailyStat[]> {
+    const clientId = getUserProperty('clientId');
+    const now = new Date().toISOString().split('T')[0];
```

Comment:
It seems dangerous to calculate the dates yourself, so why not use the dayjs library?
Please refer to my PR from brant/usage branch.

Reply:
you mean `dayjs` library?